client/components/SinglePickle.js

```diff
@@ -28,11 +28,17 @@ class SinglePickle extends Component {
         <h1>{pickle.title} </h1>
         <img src={pickle.imageUrl} />
         <p> About this pickle: {pickle.description} </p>
-        <p> Price: ${pickle.price} </p>
+        <p> Price: ${(pickle.price / 100).toFixed(2)} </p>
         <button type="button" onClick={() => this.handleClick(pickle)}>
           {' '}
           Add to cart
         </button>
+        <p>
```

Comment:
ideas for future updates on this component display: 
- display the nickname of the user/guest leaving the comment
- add fakerjs seed data that lists the content, adds associations between userId/guest(null) & pickleId
- display a "no review" div if there are no reviews associated to the pickle
- date/time or just date of review

Reply:
Yup. All sounds good. It also probably makes sense to create a Review component and render it via SinglePickle. (I added the review section just to get things started). We can work on it after the admin stuff (from tier 1) is done :)